app/gateway-oss/2.5.x/hybrid-mode.md

```diff
@@ -417,38 +417,50 @@ automatically within seconds.
 
 ### Version compatibility
 
-{{site.ce_product_name}} control planes only connect to data planes with the
-same major version and at most two minor versions earlier (edge included).
-Control planes won't connect to data planes with newer versions.
+{{site.ce_product_name}} control planes only allow connections from data planes
+with the same major version.
+Control planes won't allow connections from data planes with newer minor versions.
 
 For example, a {{site.ce_product_name}} v2.4.2 control plane:
 
 - Accepts a {{site.ce_product_name}} 2.4.0, 2.4.1 and 2.4.2 data plane
 - Accepts a {{site.ce_product_name}} 2.3.8, 2.2.1 and 2.2.0 data plane
-- Rejects a {{site.ce_product_name}} 2.4.3 data plane
-- Rejects a {{site.ce_product_name}} 2.1.9 data plane
-- Rejects a {{site.ce_product_name}} 1.0.0 data plane
+- Accepts a {{site.ce_product_name}} 2.4.3 data plane
+- Rejects a {{site.ce_product_name}} 1.0.0 data plane (major version differs)
+- Rejects a {{site.ce_product_name}} 2.5.0 data plane (minor version on data plane is newer)
 
-Furthermore, {{site.ce_product_name}} control planes only accept data planes
-whose list of plugins is a superset of the plugins installed on it. Also,
-plugins installed on both control planes and data planes, must have the same
-major and minor versions.
+Furthermore, for every plugin that is configured on the {{site.ce_product_name}}
+control planes, new configs are only pushed to data planes that have those configured
+plugins installed and loaded, and the major version of those configured plugins must
+be the same on both control planes and data planes, and minor version on the data planes
+could not be newer than versions installed on the control planes. (Configured plugins means
```

Comment:
Let's also mention patch version is ignored regardless or its value.

Reply:
Added.